lib/page-object/platforms/watir_webdriver.rb

```diff
@@ -9,7 +9,20 @@ def self.create_page_object(browser)
 
       def self.is_for?(browser)
         require 'watir-webdriver'
-        browser.is_a?(::Watir::Browser)
+        browser.is_a?(::Watir::Browser) || browser.is_a?(::Watir::HTMLElement)
+      end
+
+      def self.browser_for root
+        return root if root.is_a?(::Watir::Browser)
+        root.browser
+      end
+
+      def self.root_element_for root
+        Elements::Element.new root, :platform => :watir_webdriver if root.is_a? ::Watir::HTMLElement
+      end
+
+      def self.browser_root_for browser
+        browser.element
```

Comment:
The root html tag is just the first element in the DOM, so it can be asked for that way. I am not aware of a case where this isn't so, but if you know of one, I can address that.